### ISSUE TITLE
UX : Use more intuitive icons for rotating images

### ISSUE THREAD

**Issue by graciousgrey** [photoprism/photoprism#3643]
As a user I want more intuitive icons to rotate images, so that I am not confused.

**Acceptance Criteria:**
- Instead of the [current implementation](https://docs.photoprism.app/user-guide/organize/rotate/) use icons for "rotate left" and "rotate right"

**Comment by kevincox** [comment]
There is some discussion of other approaches in this thread: https://github.com/photoprism/photoprism/discussions/3641#discussioncomment-6741921

I think the key is that the current "select the desired EXIF flag" isn't very intuitive. I think "rotate left" and "rotate right" would be an improvement but there are definitely other options that may be even more user-friendly such as displaying the image rotated and having the user select which one they want or having the current portrait icons represent "select the current orientation" and then compute the desired EXIF flag from that.

**Comment by scarolan** [comment]
Yes please.  It's confusing to look at a photo that got imported with everybody's head pointed to the left.  Then I have to click the icon that points the head to the right to get it straight.  Rotate buttons with circular arrows would make *much* more sense.

**Comment by bigntallmike** [comment]
... agreed that it needs improvement, but it needs to be clockwise and counter (or anti)-clockwise and not "left" and "right" ... if we're improving it, it might as well be correct.

**Comment by kevincox** [comment]
The more I think about it I think the best option seems to be using the original photo as a guide.  I think it makes it very easy for the user to just click the orientation they want without needing to do any rotations in their head.

My ugly mock from the previous discussion:
![rotate-mock](https://github.com/photoprism/photoprism/assets/494012/c81f160f-b7e6-4746-adde-b0492ca19f5f)

But I also see the benefit of having two quarter turn buttons. They are small enough to just be always visible without needing to open a dropdown. It is maybe slightly harder to get the right option but even if you get it wrong just click twice more without moving the mouse. It would also be nice to have hotkeys for this like `r` (for quarter rotation, probably clockwise by default, mash to go further) or `<Shift>-<Left>` and `<Shift>-<Right>`
